kannada_metho2.py

```diff
@@ -0,0 +1,28 @@
+def kannada_method2():
+    f1=open("C:/Users/chetan/Desktop/tamil-text-summarization-master/article-1.txt","r",encoding="utf8")

```

Comment:
@handigun Can we keep this file path as a configurable parameter? Never hardcode paths / URLS